mirar/pipelines/winter/models/_diff.py

```diff
@@ -0,0 +1,76 @@
+"""
+Models for the 'diff' table
+"""
+import os
+from typing import ClassVar
+
+from pydantic import Field, validator
+from sqlalchemy import VARCHAR, Column, Double, ForeignKey, Integer, Sequence
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from mirar.database.base_model import BaseDB
+from mirar.pipelines.winter.models._exposures import Exposure
+from mirar.pipelines.winter.models.base_model import WinterBase
+
+
+class DiffsTable(WinterBase):  # pylint: disable=too-few-public-methods
+    """
+    Diffs table in database
+    """
+
+    __tablename__ = "diffs"
+    __table_args__ = {"extend_existing": True}
+
+    udiffid = Column(
+        Integer,
+        Sequence(start=1, name="diff_udiffid_seq"),
+        autoincrement=True,
+        unique=True,
+    )
+    diffid = Column(Double, primary_key=True, autoincrement=False)
+
+    uexpid: Mapped[int] = mapped_column(ForeignKey("exposures.uexpid"))
```

Comment:
Should be urawid?

Reply:
I added rawid as well. Is that better?

Reply:
Q. was do we need uexpid? But we can decide this later too, not a major concern.

Reply:
I guess not necessarily. We can remove in future if it's unused